api/src/pcapi/utils/db.py

```diff
@@ -163,3 +168,78 @@ def detect_not_valid_constraints() -> None:
         return
     names = sorted(row[0] for row in rows)
     logger.error("Found PostgreSQL constraints that are NOT VALID: %s", names)
+
+
+@blueprint.cli.command("export_pg_stat_user_indexes")
+@cron_decorators.log_cron_with_transaction
+def export_pg_stat_user_indexes() -> None:
+    """Export statistics about indexes usage."""
+    statement = """
+        SELECT
+            now() :: date,
+            relname,
+            indexrelname,
+            idx_scan,
+            idx_tup_read,
+            idx_tup_fetch
+        FROM
+            pg_stat_user_indexes
+        WHERE schemaname='public'
+        ORDER BY
+            relname,
+            indexrelname DESC;
+    """
+    res = db.session.execute(statement)
+    rows = res.fetchall()
+    _upload_as_csv_to_google_drive(
+        "pg_stat_user_indexes", ("date", "relname", "idx_scan", "idx_tup_read", "idx_tup_fetch"), rows
+    )
+    logger.info("Exported data from pg_stat_user_indexes")
+
+
+@blueprint.cli.command("export_pg_stat_user_tables")
+@cron_decorators.log_cron_with_transaction
+def export_pg_stat_user_tables() -> None:
+    """Export statistics about tables scan."""
+    statement = """
+        SELECT
+            now() :: date,
+            relname,
+            seq_scan,
+            idx_scan
+        FROM
+            pg_stat_user_tables
+        WHERE
+            schemaname = 'public'
+        ORDER BY
+            relname;
+    """
+    res = db.session.execute(statement)
+    rows = res.fetchall()
+    _upload_as_csv_to_google_drive("pg_stat_user_tables.csv", ("date", "relname", "seq_scan", "idx_scan"), rows)
```

Comment:
On ajoute déjà le suffixe `.csv` dans la fonction `_upload_as_csv_to_google_drive()`. Tel quel, ça donnerait donc un fichier nommé `pg_stat_user_tables.csv_20240604.csv`, il me semble. On devrait pouvoir juste écrire : 

```suggestion
    _upload_as_csv_to_google_drive("pg_stat_user_tables", ("date", "relname", "seq_scan", "idx_scan"), rows)
```